deeppavlov/models/torch_bert/torch_transformers_el_ranker.py

```diff
@@ -0,0 +1,353 @@
+from pathlib import Path
+from logging import getLogger
+from typing import List, Optional, Dict, Tuple, Union, Any
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import numpy as np
+from torch import Tensor
+# from apex import amp
+
+from deeppavlov.core.commands.utils import expand_path
+from transformers import AutoConfig, AutoTokenizer, AutoModel
+from deeppavlov.core.common.errors import ConfigError
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.torch_model import TorchModel
+from deeppavlov.models.preprocessors.torch_transformers_preprocessor import TorchTransformersEntityRankerPreprocessor
+
+log = getLogger(__name__)
+
+
+@register('torch_transformers_el_ranker')
```

Comment:
doesn't used as component in configuration files, remove @register

Reply:
added config configs/ranking/el_ranking_distilbert_eng.json, where this component is used